internal/pkg/action/operator_uninstall.go

```diff
@@ -140,15 +137,23 @@ func (u *OperatorUninstall) deleteObjects(ctx context.Context, objs ...controlle
 	return waitForDeletion(ctx, u.config.Client, objs...)
 }
 
-func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, installPlanKey types.NamespacedName) (crds, csvs, others []controllerutil.Object, err error) {
-	installPlan := &v1alpha1.InstallPlan{}
-	if err := u.config.Client.Get(ctx, installPlanKey, installPlan); err != nil {
-		return nil, nil, nil, fmt.Errorf("get install plan: %v", err)
+func (u *OperatorUninstall) getInstallPlanForCSV(ctx context.Context, installedCSV, ipNamespace string) (*v1alpha1.InstallPlan, error) {
```

Comment:
Nit, we know the subscription namespace is the same as `u.config.Namespace`, so I think we can simplify the signature and use `u.config.Namespace` anywhere we'd use `ipNamespace`.

```suggestion
func (u *OperatorUninstall) getInstallPlanForCSV(ctx context.Context, installedCSV) (*v1alpha1.InstallPlan, error) {
```